enzyme/test/Enzyme/ReverseMode/blas/gemm_beta_f_c.ll

```diff
@@ -0,0 +1,236 @@
+;RUN: if [ %llvmver -lt 16 ]; then %opt < %s %loadEnzyme -enzyme -S | FileCheck %s; fi
+;RUN: %opt < %s %newLoadEnzyme -passes="enzyme" -S | FileCheck %s
+
+declare void @dgemm_64_(i8* nocapture readonly, i8* nocapture readonly, i8* nocapture readonly, i8* nocapture readonly, i8* nocapture readonly, i8* nocapture readonly, i8*, i8* nocapture readonly, i8*, i8* nocapture readonly, i8* nocapture readonly, i8*, i8* nocapture readonly) 
+
+define void @f(i8* %C, i8* %A, i8* %B, i8* %beta) {
+entry:
+  %transa = alloca i8, align 1
+  %transb = alloca i8, align 1
+  %m = alloca i64, align 16
+  %m_p = bitcast i64* %m to i8*
+  %n = alloca i64, align 16
+  %n_p = bitcast i64* %n to i8*
+  %k = alloca i64, align 16
+  %k_p = bitcast i64* %k to i8*
+  %alpha = alloca double, align 16
+  %alpha_p = bitcast double* %alpha to i8*
+  %lda = alloca i64, align 16
+  %lda_p = bitcast i64* %lda to i8*
+  %ldb = alloca i64, align 16
+  %ldb_p = bitcast i64* %ldb to i8*
+  ;%beta = alloca double, align 16
+  ;%beta_p = bitcast double* %beta to i8*
+  %ldc = alloca i64, align 16
+  %ldc_p = bitcast i64* %ldc to i8*
+  store i8 78, i8* %transa, align 1
+  store i8 78, i8* %transb, align 1
+  store i64 4, i64* %m, align 16
+  store i64 4, i64* %n, align 16
+  store i64 8, i64* %k, align 16
+  store double 1.000000e+00, double* %alpha, align 16
+  store i64 4, i64* %lda, align 16
+  store i64 8, i64* %ldb, align 16
+  ;store double 0.000000e+00, double* %beta
+  store i64 4, i64* %ldc, align 16
+  call void @dgemm_64_(i8* %transa, i8* %transb, i8* %m_p, i8* %n_p, i8* %k_p, i8* %alpha_p, i8* %A, i8* %lda_p, i8* %B, i8* %ldb_p, i8* %beta, i8* %C, i8* %ldc_p) 
+  ;call void @dgemm_64_(i8* %transa, i8* %transb, i8* %m_p, i8* %n_p, i8* %k_p, i8* %alpha_p, i8* %A, i8* %lda_p, i8* %B, i8* %ldb_p, i8* %beta_p, i8* %C, i8* %ldc_p) 
```

Comment:
whats going on with the commented out lines here? Maybe add a comment explaining this or remove them.

Reply:
this test is not checking what it should check later, I just use it to trigger abi missmatches in my implementation of an inner product.